tools/ci_build/github/azure-pipelines/templates/py-packaging-stage.yml

```diff
@@ -20,15 +20,10 @@ parameters:
   type: boolean
   default: true
 
-- name: enable_linux_gpu_training_cu102
-  displayName: 'Whether Linux GPU Cuda 10.2 package is built.'
+- name: enable_linux_gpu_tests
+  displayName: 'Whether to run tests when building the Linux GPU package.'
   type: boolean
-  default: false
-
-- name: enable_linux_gpu_training_cu111
```

Comment:
removed unused parameters enable_linux_gpu_training_cu102 and enable_linux_gpu_training_cu111